source/_integrations/synologydsm.markdown

```diff
@@ -50,6 +50,10 @@ password:
   description: The password of the user to connect to the Synology NAS.
   required: true
   type: string
+api_version:
+  description: Define DSM version to allow backward compatibility with 5.x. Value can be `5` for DSM 5.x or `6` for DSM 6.x or later.
```

Comment:
As it's an optional value can you add 6 is the default ?

Reply:
I did add the default value but it has been removed since it's not defaulted in the component, but rather in the API directly... Shall I include it anyway ?

Reply:
The documentation should reflect our codebase, not that of a third party.

Reply:
Good to me, indeed after @aaska reply I thought of either: add a default in HA, or don’t precise in the doc (but still will be 6 as default).